Jenkinsfile

```
@@ -0,0 +1,23 @@
+pipeline {
+
+ options {
+  skipDefaultCheckout()
+  buildDiscarder(logRotator(numToKeepStr: '20'))
+ }
+ agent any
+ stages {
+  stage('Tests') {
+   steps {
+    script {
+     env.WORKSPACE = pwd()
+     iC = docker.image("openjdk:8-jdk")
```

```suggestion
     docker.image("openjdk:8-jdk").inside {
```
Do you run any containers during the build (e.g. IrohaContainers)? If not, you do not really need mounting Docker socket inside the container.

---

I run Iroha container inside it.